spring-cloud-stream-binder-kinesis-docs/src/main/asciidoc/overview.adoc

```diff
@@ -345,6 +346,16 @@ Here are the policies statements that your application role need:
             "arn:aws:dynamodb:<region>:<account>:table/<name-of-metadata-table>",
             "arn:aws:dynamodb:<region>:<account>:table/<name-of-lock-table>"
           ]
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "kms:GenerateDataKey",
+		"kms:Decrypt"
```

Comment:
We don't use any KMS in this project.
Why do we need to mention these permissions then?

Reply:
My bad not updating the title.

It's for the error:
> `kms:GenerateDataKey`(for producer) and `kms:Decrypt`(for consumer) should be allowed on the corresponding KMS key as well.

```
Got an exception com.amazonaws.services.kinesis.model.KMSAccessDeniedException:
User *:* is not authorized to decrypt records in stream *
(Service: AmazonKinesis; Status Code: 400; Error Code: KMSAccessDeniedException; Request ID: *; Proxy: null)
during [ShardConsumer{shardOffset=KinesisShardOffset{iteratorType=TRIM_HORIZON, sequenceNumber='null',
timestamp=null, stream='service-platform-kinesis-demo', shard='shardId-000000000000', reset=false},
state=CONSUME}] task invocation.
```

But you are right, let me remove it if KMS is not used.

Reply:
TIL: [Server-Side Encryption](https://docs.aws.amazon.com/streams/latest/dev/getting-started-with-sse.html) is optional. So I think we can make it in a separated section.

Reply:
Updated the commit. If you think the new section is unrelated and unnecessary to this project, I'll remove it.